tools/sdk/lib/fix_sdk_libs.sh

```diff
@@ -1,36 +1,67 @@
 #!/bin/bash
 set -e
 
-export PATH=../../xtensa-lx106-elf/bin:$PATH
+export PATH=../../../xtensa-lx106-elf/bin:$PATH
```

Comment:
this was correct though? from the tools/sdk/lib, we expect get.py to put toolchain there
```
> realpath -m ../../xtensa-lx106-elf/bin
/home/builder/.platformio/packages/framework-arduinoespressif8266@src-31d658a59f41540201fc3726a1394910/tools/xtensa-lx106-elf/bin
```

Reply:
This script must be run from one of the subdirectories though as it does patch the contents in the subdirectory. So how I used it (and how I think it should be used) is like this:

```
cd tools/sdk/lib/
cd NONOSDK221
../fix_sdk_libs.sh
```